src/components/graphs/DualAnnualCycleGraph.js

```diff
@@ -146,9 +143,7 @@ export default function DualAnnualCycleGraph(props) {
         // determine whether the data ranges overlap:
         const yRange = yAxisRange(graph, 'y');
         const y2Range = yAxisRange(graph, 'y2');
-        if (yAxisUnits(graph, 'y') === yAxisUnits(graph, 'y2') &&
-            (((yRange.min < y2Range.min) && (yRange.max > y2Range.min)) ||
-             ((yRange.min < y2Range.max) && (yRange.max > y2Range.max)))) {
+        if(!(yRange.max < y2Range.min || y2Range.max < yRange.min)) {
```

Comment:
Oops, think we lost units matching condition here.

Reply:
Oops. Thanks for catching that!